completion.bash

```diff
@@ -42,10 +42,28 @@ _charmcraft()
             COMPREPLY=( $(compgen -W "--help" -- "$cur") )
```

Comment:
way up there ^^^ there's a `cmds=(...)` that lists all the commands the completer knows about...

Reply:
Ah, nice, thanks